tekore/client/full.py

```diff
@@ -53,6 +53,16 @@ def token_as(self, token) -> 'Spotify':
         yield self
         self.token = old_token
 
+    def _get_paging_result(self, address: str):
+        result = self._get(address)
```

Comment:
Here `self._get` returns Request, `_get_paging_result` needs to be rewritten somehow